apis/python/tests/test_indexer.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
```

Comment:
absolutely *NOT* required for this PR, but htis file should use pytest conventions, e.g.,  pytest.parametrize, pytest.raises, etc.

Reply:
I'll make sure to create a follow up small task for that.